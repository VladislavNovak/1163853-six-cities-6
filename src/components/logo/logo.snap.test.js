import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';

import {Logo} from '..';

test(`Should Logo render correctly`, () => {
  const history = createMemoryHistory();
  const classNameForLogoLink = `usual`;

  const {container} = render(
      <Router history={history}>
        <Logo classNameForLogoLink={classNameForLogoLink}/>
      </Router>
  );
  expect(container).toMatchSnapshot();
});
