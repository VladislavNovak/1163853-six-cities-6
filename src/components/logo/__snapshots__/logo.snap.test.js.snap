// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Logo render correctly 1`] = `
<div>
  <div
    class="header__left"
  >
    <a
      class="header__logo-link usual"
      href="/"
    >
      <img
        alt="6 cities logo"
        class="header__logo"
        height="41"
        src="img/logo.svg"
        width="81"
      />
    </a>
  </div>
</div>
`;
